test/verify/check-pages

```diff
@@ -254,11 +254,11 @@ OnCalendar=daily
         b.wait_visible('#password-group')
         b.wait_text("#password-group > label", "Passwort")
 
-        # Test all languages
+    @testlib.onlyImage("Enough to test on one OS, not OS specific", "rhel-9*")
```

Comment:
Neutral question: what's the policy behind things like this?  I feel like this might cut across some other assumptions we make like "oh, I'll just run this PR against `fedora-ostree`, because this change is not OS specific" and then this test would start failing only on the next PR...

Reply:
The very formal definition of objective criteria used to be "gut feeling" :sweat_smile: I kept this here because testBasic() did the same. I'll need to change it for BiDi, but I didn't want semantic changes in *this* PR.